Traceback (most recent call last):
  File "D:\Laura Scoccianti\UniTN\Simulation and Performance Evaluation\Final Project\dtnSimulator\src\core.py", line 36, in <module>
    main()
  File "D:\Laura Scoccianti\UniTN\Simulation and Performance Evaluation\Final Project\dtnSimulator\src\core.py", line 32, in main
    launch_batch_simulations()
  File "D:\Laura Scoccianti\UniTN\Simulation and Performance Evaluation\Final Project\dtnSimulator\src\core.py", line 28, in launch_batch_simulations
    simulator.run(seconds=simulation_seconds)
  File "D:\Laura Scoccianti\UniTN\Simulation and Performance Evaluation\Final Project\dtnSimulator\src\simulator.py", line 31, in run
    map = Map(self.no_nodes, self.no_sinks, self.node_signal, \
  File "D:\Laura Scoccianti\UniTN\Simulation and Performance Evaluation\Final Project\dtnSimulator\src\map.py", line 20, in __init__
    self.network = self.generate_network(self.size)
  File "D:\Laura Scoccianti\UniTN\Simulation and Performance Evaluation\Final Project\dtnSimulator\src\map.py", line 48, in generate_network
    waypoint_matrix = numpy.full(shape=(rows_no,columns_no), fill_value=Waypoint())
  File "C:\Users\laura\AppData\Local\Programs\Python\Python39\lib\site-packages\numpy\core\numeric.py", line 314, in full
    a = empty(shape, dtype, order)
TypeError: 'float' object cannot be interpreted as an integer